en/contribute/dev_call.md

```diff
@@ -18,30 +16,19 @@ There is also space in the agenda to discuss pull requests, major impacting issu
 * Community members
 
 :::tip
-The dev call is open to all interested developers (not just the core dev team). 
+The Community Q&A call is open to all interested community members.
+
 This is a great opportunity to meet the team and contribute to the ongoing development of the platform.
 :::
 
 ## What gets discussed?
 
-The first/main part of the meeting runs for 45 minutes and provides a high-level forum to discuss where the project is going. 
-
-This is where we discuss *contributions*, including issues/PRs that have the [dev call](https://github.com/PX4/PX4-Autopilot/labels/devcall5) label.
-We expect the proposer and the assigned reviewer to be on the call!
+The meeting note gets published week before the call on [PX4 Discuss - weekly-dev-call](https://discuss.px4.io/c/weekly-dev-call).
 
-:::note
-The main call is designed to support rapid/focused decision making.
-We don't expect deep technical discussions and we will not spend extended amounts of time on feature requests. 
-Proposals are welcome, but they need a sponsor (someone willing to *implement* the work)!
-:::
-
-The second part of the meeting is for in-depth technical discussions and open ended questions.
-The core team/subsystem maintainers will be available for up to 45 additional minutes. 
+We welcome any topics that you, as a community member may have questions about / want to discuss!
 
+Please add your agendas as a reply to the meeting note before the meeting begins, as it helps clearing up confusion about your topic & will help you formulate the topic in a better manner.
```

Comment:
```suggestion
Please add your topics for discussion to the agenda before the meeting begins, by replying to the meeting note.
This will help you formulate your questions more clearly, and allow us to think about them in advance.
```